docs/release-notes.md

```diff
@@ -22,6 +22,8 @@ To upgrade from version 1.x to 2.x, contact Ondat [support](/docs/support) for a
 
 2.8.0 Release Notes
 
+> 💡 For Ondat 2.8.0, we recommend setting up your cluster to have 5 nodes if you put your etcd inside the cluster, as it's recommended to run etcd with 5 replicas (with one on each node).
```

Comment:
```suggestion
> 💡 For Ondat 2.8.0, we recommend having at least a 5 node cluster when running etcd within Kubernetes, as we recommend running etcd with 5 replicas.
```

Reply:
Is 5 nodes now considered as a minimum requirement moving forward? If so I would rather state 5 nodes instead of 3 as the minimum requirement to run Ondat to avoid confusing users down the road.

Reply:
@hubvu we should make it clear in the min requirement as well

Reply:
@hubvu 5 nodes for a "production" cluster with etcd inside. 3 nodes (with 3 etcd replicas) is fine for non-production